client.lua

```diff
@@ -163,199 +199,106 @@ local function startTestDriveTimer(testDriveTime, prevCoords)
-            insideShop = name
-            CreateThread(function()
-                while insideShop and PlayerData.job and PlayerData.job.name == Config.Shops[name]['Job'] do
-                    setClosestShowroomVehicle()
-
-                    lib.registerContext({
-                        id = 'veh_menu',
-                        title = getVehBrand():upper() .. ' ' .. getVehName():upper() .. ' - $' .. getVehPrice(),
-                        options = {
-                            {
-                                title = Lang:t('menus.test_header'),
-                                description = Lang:t('menus.managed_test_txt'),
-                                event = 'qb-vehicleshop:client:openIdMenu',
-                                args = {
-                                    vehicle = Config.Shops[insideShop]["ShowroomVehicles"][ClosestVehicle].chosenVehicle,
-                                    type = 'testDrive'
-                                }
-                            },
-                            {
-                                title = Lang:t('menus.managed_sell_header'),
-                                description = Lang:t('menus.managed_sell_txt'),
-                                event = 'qb-vehicleshop:client:openIdMenu',
-                                args = {
-                                    vehicle = Config.Shops[insideShop]["ShowroomVehicles"][ClosestVehicle].chosenVehicle,
-                                    type = 'sellVehicle'
-                                }
-                            },
-                            {
-                                title = Lang:t('menus.finance_header'),
-                                description = Lang:t('menus.managed_finance_txt'),
-                                event = 'qb-vehicleshop:client:openCustomFinance',
-                                args = {
-                                    price = getVehPrice(),
-                                    vehicle = Config.Shops[insideShop]["ShowroomVehicles"][ClosestVehicle].chosenVehicle
-                                }
-                            },
-                            {
-                                title = Lang:t('menus.swap_header'),
-                                description = Lang:t('menus.swap_txt'),
-                                event = 'qb-vehicleshop:client:vehCategories',
-                                arrow = true
-                            },
-                        }
-                    })
-                    Wait(1000)
-                end
-            end)
-        else
-            insideShop = nil
-            ClosestVehicle = 1
-        end
-    end)
+local function enteringFinancingZone()
+    lib.showTextUI(Lang:t('menus.keypress_showFinanceMenu'))
+end
+
+local function insideFinancingZone()
+    if IsControlJustPressed(0, 38) then
+        lib.showContext('fin_header_menu')
+    end
+end
+
+local function exitingFinancingZone()
+    lib.hideTextUI()
 end
```

Comment:
Also a useless function, you can just use lib.hideTextUI() directly instead of putting it in a function, it does nothing special that is hard to write yourself or dynamic

Reply:
done.